pkg/processor/native/processor_test.go

```diff
@@ -62,6 +63,68 @@ func (s *BildProcessorSuite) TestBildProcessor_Grayscale() {
 	assert.EqualValues(s.T(), actual, expected)
 }
 
+func (s *BildProcessorSuite) TestBildProcessor_Flip() {
```

Comment:
can you refactor this into defining cases array and looping that instead of asserting one by one like this?
you can refer to this part of code:
https://github.com/gojek/darkroom/blob/df6feb067885d4703f77803b58c228fd8447dc8b/pkg/config/config_test.go#L10-L34